path_config.go

```diff
@@ -227,6 +233,12 @@ func (b *jwtAuthBackend) pathConfigWrite(ctx context.Context, req *logical.Reque
 		}
 	}
 
+	switch config.OIDCResponseMode {
+	case "", "query", "form_post":
```

Comment:
Not essential, but it might be worth considering using an enum for these. I use enums to be more defensive because it reduces the possibility of typos and makes it clearer what is and isn't supported. However, on the downside, sometimes an enum can add up to _more_ code. So definitely a judgement call.